subsys/bluetooth/controller/Kconfig

```diff
@@ -121,38 +121,33 @@ config BT_CTLR_SUBRATING_SUPPORT
 config BT_CTLR_CHANNEL_SOUNDING_SUPPORT
 	bool
 
+# Workaround to have commas on function arguments
+ZEPHYR_BT_HCI := zephyr,bt-hci
+
 config BT_CTLR
-	bool "Bluetooth Controller"
+	bool
 	help
 	  Enables support for SoC native controller implementations.
 
-if BT_CTLR
-
-choice BT_LL_CHOICE
-	prompt "Bluetooth Link Layer Selection"
-	help
-	  Select the Bluetooth Link Layer to compile.
-	  The link layer choice list can be extended out of tree.
-	  In the nRF Connect SDK, the SoftDevice Controller is selected
-	  and set as the default Bluetooth Controller.
-
 config BT_LL_SW_SPLIT
 	bool "Software-based BLE Link Layer"
+	default y
 	depends on DT_HAS_ZEPHYR_BT_HCI_LL_SW_SPLIT_ENABLED
+	select BT_CTLR
 	help
 	  Use Zephyr software BLE Link Layer ULL LLL split implementation.
 
-endchoice
-
-config BT_CTLR_HCI
-	bool "Host Controller Interface (HCI)"
+config BT_LL_SW_SPLIT_HCI
+	bool "Host Controller Interface (HCI) for the split link layer"
 	default y
-	depends on DT_HAS_ZEPHYR_BT_HCI_LL_SW_SPLIT_ENABLED
+	depends on BT_LL_SW_SPLIT
```

Comment:
This is open to discussion on how to include downstream Controllers/Blobs (with/without HCI layer)